### PR TITLE
Add rule elements for Brute Strength and Trident Bully

### PR DESCRIPTION
**Pull request by simonward**
~~Blocked by https://github.com/foundryvtt/pf2e/pull/8082~~

### CODE REVIEW

File: packs/data/strength-of-thousands-bestiary.db/abendego-brute.json [hunk 1]
```diff
@@ -484,12 +483,32 @@
...
+                        "key": "DamageDice",
+                        "predicate": [
+                            "item:base:trident"
+                        ],
+                        "selector": "strike-damage"
+                    },
+                    {
+                        "definition": [
+                            "item:base:trident",
+                            "item:melee"
```
Comment (stwlam) [inline packs/data/strength-of-thousands-bestiary.db/abendego-brute.json:504]:
```suggestion
```

File: packs/data/strength-of-thousands-bestiary.db/abendego-brute.json [hunk 2]
```diff
@@ -484,12 +483,32 @@
...
+                        "dieSize": "d8",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "item:base:trident"
+                        ],
+                        "selector": "strike-damage"
+                    },
+                    {
+                        "definition": [
+                            "item:base:trident",
```
Comment (stwlam) [inline packs/data/strength-of-thousands-bestiary.db/abendego-brute.json:503]:
```suggestion
                            "item:base:trident"
```

File: packs/data/strength-of-thousands-bestiary.db/abendego-brute.json (hunk 1 above)
Comment (simonward) [reply packs/data/strength-of-thousands-bestiary.db/abendego-brute.json:504]: The description says "... a trident gains the shove trait while the brute is using it in melee..."
Is the melee predicate superfluous here?

Comment (stwlam) [reply packs/data/strength-of-thousands-bestiary.db/abendego-brute.json:504]: The shove trait isn't usable at range even when a weapon always has it

**Comment by stwlam** [comment]
It looks like Brute Strength is functionally identical but just slower to load now

**Comment by simonward** [comment]
> It looks like Brute Strength is functionally identical but just slower to load now

True, although this is technically more correct if the NPC gets hold of another trident. That said, I'm happy to revert that portion if you prefer it the old way.

**Comment by stwlam** [comment]
I'll defer to the data coordinators on that.

File: packs/data/strength-of-thousands-bestiary.db/ajbal-kimon.json [hunk 3]
```diff
@@ -882,7 +881,30 @@
...
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "criticalSuccess"
+                        ],
+                        "predicate": [
+                            "item:base:trident"
+                        ],
+                        "selector": "strike-attack-roll",
+                        "text": "PF2E.NPCAbility.AjbalKimon.TridentBullyNote",
```
Comment (TikaelSol) [inline packs/data/strength-of-thousands-bestiary.db/ajbal-kimon.json:904]:
```suggestion
                        "text": "{item|system.description.value}",
```
We don't need to localize stuff just to chop off a tiny bit of the description.

File: static/lang/re-en.json [hunk 4]
```diff
@@ -537,6 +537,9 @@
             }
         },
         "NPCAbility": {
+            "AjbalKimon": {
+                "TridentBullyNote": "Any creature Ajbal critically hits with a trident is @UUID[Compendium.pf2e.conditionitems.AJh5ex99aV6VTggg]{Flat-Footed} until the start of Ajbal's next turn."
+            },
             "AncientWisp": {
```
Comment (TikaelSol) [inline static/lang/re-en.json:543]:
```suggestion
        "NPCAbility": {
            "AncientWisp": {
```

**Pull request merged**